### PR TITLE
Updates to fed docs for 0.43.0

### PR DESCRIPTION
**Pull request by lkysow**

### CODE REVIEW

File: website/content/docs/k8s/installation/multi-cluster/kubernetes.mdx [hunk 1]
```diff
@@ -224,7 +239,7 @@ After the installation into your primary cluster you will need to export
 this secret:
 
 ```shell-session
-$ kubectl get secret consul-federation --output yaml > consul-federation-secret.yaml
+$ kubectl get secret consul-federation -n consul --output yaml > consul-federation-secret.yaml
```
Comment (david-yu) [inline website/content/docs/k8s/installation/multi-cluster/kubernetes.mdx:242]:
```suggestion
$ kubectl get secret consul-federation --namespace consul --output yaml > consul-federation-secret.yaml
```

File: website/content/docs/k8s/installation/multi-cluster/kubernetes.mdx [hunk 2]
```diff
@@ -403,7 +423,7 @@ To verify that both datacenters are federated, run the
 `consul members -wan` command on one of the Consul server pods:
 
 ```shell-session
-$ kubectl exec statefulset/consul-server -- consul members -wan
+$ kubectl exec statefulset/consul-server -n consul -- consul members -wan
```
Comment (david-yu) [inline website/content/docs/k8s/installation/multi-cluster/kubernetes.mdx:426]:
```suggestion
$ kubectl exec statefulset/consul-server --namespace consul -- consul members -wan
```

File: website/content/docs/k8s/installation/multi-cluster/kubernetes.mdx [hunk 3]
```diff
@@ -421,7 +441,7 @@ each datacenter can read each other's services. In this example, our `kubectl`
 context is `dc1` and we're querying for the list of services in `dc2`:
 
 ```shell-session
-$ kubectl exec statefulset/consul-server -- consul catalog services -datacenter dc2
+$ kubectl exec statefulset/consul-server -n consul -- consul catalog services -datacenter dc2
```
Comment (david-yu) [inline website/content/docs/k8s/installation/multi-cluster/kubernetes.mdx:444]:
```suggestion
$ kubectl exec statefulset/consul-server --namespace consul -- consul catalog services -datacenter dc2
```

**Review by david-yu — APPROVED** [review]

**Comment by hc-github-team-consul-core** [comment]
:cherries::white_check_mark: Cherry pick of commit 24e573edc87a8c59a72bd9578e2441d50943e790 onto `stable-website` succeeded!

**Comment by hc-github-team-consul-core** [comment]
:cherries::white_check_mark: Cherry pick of commit 24e573edc87a8c59a72bd9578e2441d50943e790 onto `release/1.12.x` succeeded!

**Pull request merged**